### PR TITLE
add nginx config to gateway deployment pattern

### PR DESCRIPTION
**Pull request by svrnm**
Since I was building a tutorial where I wanted to point towards the load balancing capabilities, I thought I add my nginx config here as an example

+ some minor fixes & improvements.

cc @mhausenblas

### CODE REVIEW

**Review by mhausenblas — APPROVED** [review]
LGTM, thanks!

File: content/en/docs/collector/deployment/gateway.md [hunk 1]
```diff
@@ -30,14 +31,63 @@ Let's have a look at such a case where we are using the load-balancing exporter:
...
 
 {{% alert title="Note" color="info" %}} Currently, the load-balancing exporter
 only supports pipelines of the `traces` type. {{% /alert %}}
 
-## Example
+## Examples
+
+### NGINX as "out-of-the-box" load balancer
+
+Assuming you have three collectors (`collector1`, `collector2` and `collector3`)
```
Comment (chalin) [inline content/en/docs/collector/deployment/gateway.md:45]:
```suggestion
Assuming you have three collectors (`collector1`, `collector2`, and `collector3`)
```

File: content/en/docs/collector/deployment/gateway.md [hunk 2]
```diff
@@ -30,14 +31,63 @@ Let's have a look at such a case where we are using the load-balancing exporter:
...
-1. The collectors are configured to send telemetry data to one or more backends.
+3. The collectors are configured to send telemetry data to one or more backends.
 
 {{% alert title="Note" color="info" %}} Currently, the load-balancing exporter
 only supports pipelines of the `traces` type. {{% /alert %}}
 
-## Example
+## Examples
+
+### NGINX as "out-of-the-box" load balancer
```
Comment (chalin) [inline content/en/docs/collector/deployment/gateway.md:43]: Consider one of the following suggestions:

```suggestion
### NGINX -- an "out-of-the-box" load balancer
```

```suggestion
### NGINX as an "out-of-the-box" load balancer
```

**Review by chalin — APPROVED** [review]
Minor comments inline. Otherwise LGTM

**Pull request merged**